client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { trim, isEmpty } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import Gravatar from 'components/gravatar';
+import FollowButton from 'reader/follow-button';
+import Gridicon from 'gridicons';
+
+const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( '/', '' );
```

Comment:
Do we need to handle site URLs not at the root of the domain?

`http://www.example.com/okapi` would end up as `www.example.comokapi`.